Add <matias> to Contributors list

cuervo357 opened the pull request:
First contribution

cuervo357 committed:
Add <matias> to Contributors list

Roshanjossey commented:
@cuervo357, could you please add your name to Contributors.md rather than CONTRIBUTING.md?

cuervo357 committed:
fixed contributing

Fixed my mistake. I changed contributing.md , i should have changed contributors.md

Roshanjossey committed:
Merge branch 'master' into add-matias

Roshanjossey commented:
@cuervo357, I'm quite elated about your pull request. I wanna evolve this project to addresses various problems faced by first-time contributors. I'd love to learn about your journey in open source community, the problems, pain points you had etc.
Could you explain how you felt when you went through the tutorial, made a pull request and learned that I merged it?

 We’ve recently added social share  to our web app. Could you please go to https://roshanjossey.github.io/first-contributions/#social-share and share your first contribution to open source?  Also, check out projects with easy issues while you’re there.
